paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -167,7 +168,8 @@ private fun SemanticsNode.accessibilityText(): String? {
     role,
     disabled,
     onClickLabel,
-    heading
+    heading,
+    toggleableState
```

Comment:
I think this can go below `selected,` to match the order TalkBack uses for the accessibility text